### ISSUE TITLE
[BUG] AttributeError: 'ProximityStump' object has no attribute 'get_exemplars'

### ISSUE THREAD

**Issue by CaioUeno** [alan-turing-institute/sktime#598]
**Describe the bug**

I'm trying to run a cross validation using ProximityStump classifier, but I get this error.
I was looking at the code and I guess that the problem is at this line:
https://github.com/alan-turing-institute/sktime/blob/3f132219e5217ba3dba4b0237f15ea847a84d5f5/sktime/classification/distance_based/_proximity_forest.py#L814
The "name" get_exemplars is replaced to pick_exemplars. Should it be changed? 
Looking at ProximityTree it stays as it is:
https://github.com/alan-turing-institute/sktime/blob/3f132219e5217ba3dba4b0237f15ea847a84d5f5/sktime/classification/distance_based/_proximity_forest.py#L1061

**To Reproduce**

```python
from sklearn.model_selection import cross_val_predict

cross_val_predict(base_model, X, y, cv=..., ...)
```
I am using ProximityStump as "base_model". X and y are from Chinatown dataset.
**Expected behavior**

It should work as fine as other sktime classifiers - ProximityForest and ProximityTree work great!
**Additional context**
 
I am working alongside Prof. Diego Furtado Silva in a project about Meta-Learning.

**Versions**
<details>

pip: 20.0.2
   setuptools: 44.0.0
      sklearn: 0.24.0
       sktime: 0.5.1
  statsmodels: 0.12.1
        numpy: 1.19.4
        scipy: 1.6.0
       Cython: None
       pandas: 1.2.0
   matplotlib: None
       joblib: 1.0.0
        numba: 0.52.0
     pmdarima: None
      tsfresh: None

</details>

**Comment by mloning** [comment]
Hi @CaioUeno - thanks for raising the issue. The distance-based model requires some work,  @TonyBagnall is keeping a list here #596. 

If you're interested in contributing, we'd really appreciate a PR to fix this!

**Comment by TonyBagnall** [comment]
closed because this is rolled into #596
